IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/ViewControllers/AddPersonViewController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AddPersonViewController.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/22/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class AddPersonViewController: UIViewController {
+
+    @IBOutlet weak var txtFieldName: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    @IBAction func btnAddPersonTouchUpInsided(_ sender: Any) {
```

Comment:
Even if we have a MVC approach right now, controllers shouldn't be responsible of handling CoreData's context.

We should have a `PersonService` and this controller should delegate to it.